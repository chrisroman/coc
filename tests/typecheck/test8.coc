(\a: *)(\f: [_1: a]a)(\x: a)(f x)
