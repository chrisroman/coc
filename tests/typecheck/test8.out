(λa: *)(λf: [_1: a]a)(λx: a)(f x)
Type: [a: *][f: [_1: a]a][x: a]a
